docs/config-file-ort-yml.md

```diff
@@ -235,3 +235,49 @@ resolutions:
     reason: "LICENSE_ACQUIRED_EXCEPTION"
     comment: "Commercial Qt license for the project was purchased, for details see https://jira.example.com/issues/SOURCING-5678"
 ```
+
+## License Choices
+
+### When to Use License Choices
+
+For multi-licensed dependencies a specific license can be selected.
+The license choice can be applied to a package.
+A choice is only valid for licenses combined with the SPDX operator `OR`.
+The given choice will be applied to concluded, detected and declared licenses.
+
+### License Choice by Package
+
+To select a license from a multi-licensed dependency an SPDX expression must be `given` as well as a `choice`.
+If the `choice` only refers to part of the license a `sub-expression` of that licenses can be given, and a `choice` will
+only be applied to the `sub-expression`.
+
+e.g.
+```yaml
+license_choices:
+  package_license_choice:
+  - package_id: package-id
+    license_choices:
+    - given: A OR B
+      choice: A
+    - given: (C OR D) AND E
+      choice: C
+      sub_expressions: C OR D
+    # Alternative to the choice above.
```

Comment:
Better make a second license_choice element; I find the in-lined alternative a bit confusing.

Reply:
I added a separate element and tried to clarify the comment.